odml/info.json

```diff
@@ -1,5 +1,5 @@
 {
-  "VERSION": "1.4.5",
+  "VERSION": "1.4.6",
   "FORMAT_VERSION": "1.1",
```

Comment:
Strictly speaking I do not think a minor version bump is required, since the changes introduced here are not breaking older libraries to open new files.

But since other changes introduced in the current master branch actually introduce breaking changes (respecting `section.type` to be a required attribute), might as well do it here lets we forget.

Reply:
hm not sure if I agree.  older libraries can read but upon opening and rewriting this information would get lost. Further, a user might expect to be able to validate a newer file but the info will be ignored.
I am fine to accept either decision,  maybe @achilleas-k can be the deciding voice ;)

Reply:
Well Mich said we're bumping it anyway for the other changes, so it doesn't matter.  Bumpybump.

Reply:
sorry @mpsonntag I only read the first part